plugins/modules/net_tools/nmcli.py

```diff
@@ -342,6 +342,14 @@
             - Name of the Wireless router or the access point.
        type: str
        version_added: 3.0.0
+    wifi:
+       description:
+            - 'The configuration of the Wifi connection. The valid attributes are listed on:'
+            - 'U(https://developer.gnome.org/NetworkManager/stable/settings-802-11-wireless.html)'
+            - 'For instance to create a hidden AP mode Wifi connection:'
+            - '- C({hidden: true, mode: ap})'
```

Comment:
```suggestion
            - 'C(- {hidden: true, mode: ap})'
```

Reply:
Thanks for the tip, I just took the structure from the `wifi_sec` section, so perhaps it also could use some adjustment.